frontend/src/loadPostHogJS.tsx

```diff
@@ -27,6 +27,10 @@ export function loadPostHogJS(): void {
                 bootstrap: !!window.POSTHOG_USER_IDENTITY_WITH_FLAGS ? window.POSTHOG_USER_IDENTITY_WITH_FLAGS : {},
                 opt_in_site_apps: true,
                 loaded: (posthog) => {
+                    if (posthog.webPerformance) {
```

Comment:
is this leftover from debugging?

Reply:
No this is a fix that I forgot to add after we made network recording disabled by default on localhost

Reply:
Would be good to display something in the UI for these cases...